post_processing/rayleigh_diagnostics.py

```diff
@@ -950,7 +950,7 @@ def read_data(self,qcodes = []):
         if (len(qcodes) == 0):
             self.vals[:,:] = fdata['fdata']['vals'][0,:,:]
         else:
-            nqfile = self.nq        # number of quantity codes in the file
+            # nqfile = self.nq        # number of quantity codes in the file
```

Comment:
We just use `self.nq` here. Let's keep just the comment.
```suggestion
            # number of quantity codes in the file
```